src/header_dialog.js

```diff
@@ -1708,6 +1708,13 @@ export function HeaderDialog(dialog, onSave) {
       setParameter("rate_limits_pitch", sysConfig.rate_limits[1], 0);
       setParameter("rate_limits_yaw", sysConfig.rate_limits[2], 0);
       $("#rate_limits").show();
+
+      if (semver.lt(activeSysConfig.firmwareVersion, "4.6.0")) {
+        const derivativeColumn = document.getElementById("derivativeColumn");
+        const dMaxColumn = document.getElementById("dMaxColumn");
+        const parent = derivativeColumn.parentNode;
+        parent.insertBefore(dMaxColumn, derivativeColumn); // Меняем местами
```

Comment:
Rename `dMaxColumn` to `DMin` here too?

Reply:
but it would be more than that...
4.5:
![image](https://github.com/user-attachments/assets/c9e2f42e-b6fe-4482-8b79-6d146fdf6635)

agree, just a matter of labels could fix.

Reply:
master configurator uses DMax verbiage in 4.5, so maybe this is fine as-is
![image](https://github.com/user-attachments/assets/a753845f-8e86-4500-90fa-69ed3cb3ba4a)